pkg/asset/machines/machineconfig/workload-partitioning.go

```diff
@@ -0,0 +1,127 @@
+package machineconfig
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	ini "gopkg.in/ini.v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/installer/pkg/asset/ignition"
+	"github.com/pkg/errors"
+)
+
+// CrioWorkloadDropinContents generates the content expected by Cri-O for workload partitioning
+//
+// Example output:
+// [crio.runtime.workloads.management]
+// label = "management.workload.openshift.io/cores"
+// annotation_prefix = "io.openshift.workload.management"
+// resources = { "cpu" = "", "cpuset" = "0-1", }
+func CrioWorkloadDropinContents(name, cpuset string) (string, error) {
+	/*
+		[crio.runtime.workloads.management]
+		label = "management.workload.openshift.io/cores"
+		annotation_prefix = "io.openshift.workload.management"
+		resources = { "cpu" = "", "cpuset" = "0-1", }
+	*/
+	crioIni := ini.Empty()
```

Comment:
The format we use is toml. CRI-O uses "github.com/BurntSushi/toml" to read it.

Reply:
Looked a lot like ini, but sure, I can switch over :)

Reply:
@mrunalp : I have a pending change that switches over to `github.com/BurtSushi/toml` and the encoder wants to create a file with this contents... Is it okay?  It looks a bit different than I was expecting:
```
["crio.runtime.workloads.master"]
label = "master.workload.openshift.io/cores"
annotation_prefix = "io.openshift.workload.master"
resources = "{ \"cpu\" = \"\", \"cpuset\" = \"0-1\", }"
```

Reply:
@mrunalp : A slack conversation with @haircommander revealed that the format produced by `github.com/BurntSushi/toml` doesn't work, but the format produced in this current PR by `gopkg.in/ini.v1` works just fine:
```
[crio.runtime.workloads.management]
label             = "management.workload.openshift.io/cores"
annotation_prefix = "io.openshift.workload.management"
resources         = { "cpu" = "", "cpuset" = "0-1", }
```
So I think it's best to stick with what I've got here - `gopkg.in/ini.v1` is already used elsewhere in this repo anyway.